README.md

```diff
@@ -5,7 +5,7 @@ Contains a script to set up a Clearpath robot computer for ROS 2 Humble or Jazzy
 This full script includes setting of ROS 2 and Clearpath package servers, installing ROS 2 and Clearpath Robot packages and installing the Clearpath Robot Service.
 
 Additional Scripts that are run as part of the main script but can also be run independently:
-`cockpit_installer.sh` - To install cockpit and the ros2 plugin
+`cockpit_installer.sh` - To install cockpit and the ros2 plugin (available for Noble / Jazzy only)
```

Comment:
For clarity, since not everyone will necessarily be familiar with Ubuntu and ROS codenames, is it worth being explicit and saying "Available for Ubuntu 24.04 and ROS 2 Jazzy only?" It's wordier, but with the "/" I can see someone getting confused and thinking that either Jazzy is an Ubuntu release, or Noble is a ROS release.